style-dark.scss

```diff
@@ -1,10 +1,10 @@
 /*-- scss:defaults --*/
 @import 'https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible&display=swap';
-@import 'https://fonts.googleapis.com/css2?family=Fira+Code:wght@300;400;500;600;700&display=swap';
+@import 'https://fonts.googleapis.com/css2?family=DM+Mono:wght@300;400;500;600;700&display=swap';
```

Comment:
Why the font switch?

Reply:
No reason but upon further reflection I think Fira looks better here.